tests/e2e/e2e_common.go

```diff
@@ -223,6 +223,38 @@ const (
 		"/api-tokens/authorize"
 )
 
+// test suite labels
+const (
+	slow                = "slow"
```

Comment:
can we add one line comment for each of the lables ,so that tomorrow even developers can check this labels and use them

Reply:
The self-explanatory Labels like block, file, tkg,wcp etc. those I am not adding any comments.. only for specific labels like slow, deprecated etc. for these I will add one line comment

ACK